docs/managed-datahub/subscription-and-notification.md

```diff
@@ -14,6 +14,19 @@ With this feature, you can set up subscriptions to specific changes for an Entit
 This feature is especially useful in helping you stay on top of any upstream changes that could impact the assets you or your stakeholders rely on. It eliminates the need for you and your team to manually check for upstream changes, or for upstream stakeholders to identify and notify impacted users.
 As a user, you can subscribe to and receive notifications about changes such as deprecations, schema changes, changes in ownership, assertions, or incidents. You’ll always been in the know about potential data quality issues so you can proactively manage your data resources.
 
+
+## Platform Admin Notifications
+
+Datahub provides three levels of notifications: platform-level, group-level, and user-level. The group and user -level notifications are 
```

Comment:
Because this page is text-heavy, might want to break this out into a bulleted list. Visual structure might help catch people's attention and make it easier to digest:
<img width="1039" alt="image" src="https://github.com/datahub-project/datahub/assets/159848059/8d7d0f36-153a-4b2a-a52d-1bad058d9bd9">